codice/ddf
# DDF-5371 Adds beta-admin-ui

#### What does this PR do?
 - Fixes a few typescript issues we had.
 - Adds a beta-admin-ui (dependency on @connexta/kanri).
 - Leaves the old admin ui around, so that people can continue to use that one if they choose.
 - Adds a link in the old admin ui so people can find the beta admin ui.
 - Note that @connexta/kanri allows extensions if necessary.

#### How should this be tested?
Verify the old admin ui works as expected.  Verify catalog ui works as expected.

Follow the link to the new beta admin ui in the old admin ui and give it a whirl.

#### Any background context you want to provide?
More context at https://github.com/connexta/kanri and see commit message.  Both places have most of the details.

Main motivation is removing our dependencies on Backbone / Marionette / etc., and making it easier to maintain and build upon.

#### What are the relevant tickets?
Fixes: #5371 

#### Screenshots
<img width="1660" alt="Screen Shot 2019-09-24 at 1 48 44 PM" src="https://user-images.githubusercontent.com/11984853/65549483-81947780-ded2-11e9-8e0b-73bcc00e052e.png">
<img width="1657" alt="Screen Shot 2019-09-24 at 1 51 33 PM" src="https://user-images.githubusercontent.com/11984853/65549484-81947780-ded2-11e9-9ab6-9a774827b484.png">


#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
